install_config/install/host_preparation.adoc

```diff
@@ -94,15 +94,22 @@ Alternatively, disable all repositories:
 +
 Note that this could take a few minutes if you have a large number of available repositories
 
-. Enable only the repositories required by {product-title} 3.7:
+. Enable only the repositories required by {product-title} 3.9:
```

Comment:
@mburke5678 this should have been a clear clue that you are merging 3.9 changes into the 3.7 branch!

There are more issues further on.